tests/e2e/tests/pilot/externalservice_test.go

```diff
@@ -58,37 +66,150 @@ func TestServiceEntry(t *testing.T) {
 			url:               "http://bing.com",
 			shouldBeReachable: false,
 		},
-		// See issue https://github.com/istio/istio/issues/7869
-		//{
-		//	name:              "REACHABLE_wikipedia.org_over_cidr_range",
-		//	config:            "testdata/networking/v1alpha3/service-entry-tcp-wikipedia-cidr.yaml",
-		//	url:               "https://www.wikipedia.org",
-		//	shouldBeReachable: true,
-		//},
-		//{
-		//	name:              "UNREACHABLE_google.com_over_cidr_range",
-		//	config:            "testdata/networking/v1alpha3/service-entry-tcp-wikipedia-cidr.yaml",
-		//	url:               "https://google.com",
-		//	shouldBeReachable: false,
-		//},
-		{
-			name:              "REACHABLE_en.wikipedia.org_over_wikipedia_wildcard",
-			config:            "testdata/networking/v1alpha3/wildcard-tls-wikipedia.yaml",
-			url:               "https://en.wikipedia.org/wiki/Main_Page",
+		{
+			name:              "UNREACHABLE_bing.com_over_bing_wildcard_443",
+			config:            "testdata/networking/v1alpha3/service-entry-wildcard-bing.yaml",
+			url:               "https://www.bing.com",
+			shouldBeReachable: false,
+		},
+		{
+			name:              "REACHABLE_wikipedia.org_over_cidr_range",
```

Comment:
this needs to be disabled. This was becoming flakey

Reply:
or lets keep it and disable it when it becomes flakey

Reply:
I was not sure which site to switch to that I know a CIDR range so I kept this one in.